config.json

```diff
@@ -1039,6 +1039,17 @@
           "transforming"
         ]
       },
+      {
+        "slug": "word-search",
+        "name": "Word Search",
+        "uuid": "e88da49b-37d7-4f84-ad64-765fb464a721",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
```

Comment:
I would increase the difficulty to at least  5

Reply:
Agreed. Done.